WooCommerce/src/androidTest/kotlin/com/woocommerce/android/AppPrefsTest.kt

```diff
@@ -611,4 +611,16 @@ class AppPrefsTest {
             )
         ).isFalse
     }
+
+    @Test
+    fun givenTTPWasUseAtLeastOnceNeverInvokeThenIsTTPWasUsedAtLeastOnceReturnsFalse() {
```

Comment:
```suggestion
    fun givenTTPWasUsedAtLeastOnceNeverInvokedThenIsTTPWasUsedAtLeastOnceReturnsFalse() {
```
Should it be like this ☝🏼 ?